Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/PromptCodeReceiver.cs

```diff
@@ -41,7 +40,13 @@ public string RedirectUri
             var authorizationUrl = url.Build().ToString();
 
             Logger.Debug("Open a browser with \"{0}\" URL", authorizationUrl);
-            Process.Start(authorizationUrl);
+#if NETSTANDARD
+            Console.WriteLine("Please visit the following URL in a web browser, then enter the code:");
```

Comment:
We should probably think about better wording than "enter the code"... it's not clear which code is meant.